examples/expectations/column_aggregate_expectation_template.py

```diff
@@ -61,6 +61,35 @@ class ExpectColumnAggregateToMatchSomeCriteria(ColumnExpectation):
     # This dictionary contains default values for any parameters that should have default values.
     default_kwarg_values = {}
 
+    def validate_configuration(self, configuration: Optional[ExpectationConfiguration]):
```

Comment:
Gonna need `from great_expectations.exceptions import InvalidExpectationConfigurationError` at the top of the file